src/screens/wallet/settings/settings.styles.ts

```diff
@@ -0,0 +1,15 @@
+import { StyleSheet } from 'react-native';
+
+import { formatSize } from '../../../styles/format-size';
+
+export const SettingsStyles = StyleSheet.create({
```

Comment:
We use `createUseStyles` function

Reply:
I didn't write this code, I just copied it from the previous component which I removed :)